app/posts/find-teacher-training/2023-04-18-removing-school-placement-vacancies.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Removing school placement vacancies
+description: We removed the school placement vacancy status as this was misleading for candidates
+date: 2023-04-18
+tags:
+  - school placements
+  - vacancies
+related:
+  items:
+    - text: Updating course vacancies
+      href: /publish-teacher-training-courses/updating-course-vacancies/
+---
+
+When a provider publishes a course, they can list all the schools where a candidate can gain experience. Previously, this list also showed whether there was a vacancy at the school.
+
+The vacancy status was a simple ‘yes’ or ‘no’ and required the provider to keep the course current.
+
+We removed the vacancy status because:
+
+- candidates do not apply to a vacancy at a school
+- providers often do not know the number of vacancies at their partner schools until late in the recruitment cycle
+- the number of vacancies can change during the recruitment cycle because candidates may need to withdraw
+- we are providing false hope that candidates will be placed at certain schools
+- providers do not always consider a candidate’s school placement preference on the application, and some don’t know about the preference
+- the accuracy of vacancy information depends on providers frequently updating their courses
+- Apply for teacher training (Apply) is going to remove the preference for placement schools
+
+## What we changed
+
+We have changed the:
+
+- school placements section of the course details
+- warning text when a course is closed
+- vacancies filter on the course list
+
+### School placements section
+
+We replaced the ‘Schools’ table with a list of schools and removed each school’s vacancy status.
+
+![School placements](school-placements.png "School placements")
+
+### Warning text
+
+If a course is closed, it is still available on Find, but candidates cannot apply.
+
+For all closed courses, we show a message:
+
+> You cannot apply for this course as it is closed for applications. To find courses accepting applications, change your search to ‘Only show courses accepting applications’.
+
+![Course closed for applications](warning-message.png "Course closed for applications")
+
+### Vacancies filter
+
+If a course is open, it is available on Find, and candidates can apply.
+
+We replaced the ‘Vacancies’ filter with one labelled ‘Applications open’. The filter is selected by default, so we ‘Only show courses accepting applications.’
+
+We show all ‘Open’ and ‘Closed’ courses if the filter is deselected.
```

Comment:
@defong do you mean there is a disconnect between the `Open` and `Closed` states and the `Applications open` filter, which has the single option `Only show courses accepting applications`? What would be a better filter?

Reply:
Seems related to
https://github.com/DFE-Digital/bat-design-history/pull/883#discussion_r1180779642

So if the course can not be applied to is the course still `Only show courses accepting applications`

Reply:
> So if the course can not be applied to is the course still `Only show courses accepting applications`

If the course cannot be applied to, it will not show in the courses list if the user has checked `Only show courses accepting applications` in the filters. Only courses open for applications are shown in the list.

This is the same interaction as the old vacancies filter where checking `Only show courses with vacancies` only shows courses open for applications in the list.

If the user knows the course URL and goes directly to a closed course, or has unchecked the filter, the apply button is replaced by the closed for applications message.